src/renderer/app/index.tsx

```diff
@@ -20,7 +20,6 @@ log.variables.env = process.env.NODE_ENV;
 log.transports.console.format = '{y}/{m}/{d} {h}:{i}:{s}.{ms} [{env}#{version}]-{processType} [{level}] > {text}';
 log.transports.console.useStyles = true;
 
-Object.assign(console, log.functions);
```

Comment:
Please revert this change
For dev purposes you can comment this line but don't forget to put it back

Reply:
Oh sorry, ofc